docs/SQUASHING.md

```diff
@@ -0,0 +1,59 @@
+# How do I "squash" my commits?
+
+Before you start, [make sure your text editor is associated with Git](https://help.github.com/articles/associating-text-editors-with-git/) to not close without a response, otherwise you'll see some weird behavior.
+
+Then update your refs to be sure you have the latest code:
+
+```shell
+ git fetch --all
+```
+
+## Rebase
+
+Rather than merge develop into your branch, we prefer to rebase off of develop. Depending on how you are working with the repo, this step may be different.
+
+If you are working from a forked copy run:
+
+```shell
+git rebase -i develop
```

Comment:
Here we could also do `git rebase -i HEAD~X`, X being the number of commits that have been done. As far as I know, this is functionally the same, just fancier and more likely to inspire the admiration of other git users 😛